soymsg/pomsg/pomsg.go

```diff
@@ -92,7 +109,20 @@ func Dir(dirname string) (soymsg.Provider, error) {
 }
 
 func (p provider) Bundle(locale string) soymsg.Bundle {
```

Comment:
This locale is guaranteed to be one of the ones passed to Load, right? 

In that case, it might be better to keep the mapping of "requested locale" to "actual locale" in the provider, rather than calculating fallback in two different places?

Reply:
That makes sense, I think the bundler is the only thing that _truly_ cares about fallbacks, everything else is largely redundancies. Putting up a patch now.